kucoin.go

```diff
@@ -365,6 +365,44 @@ func (b *Kucoin) CreateOrder(symbol, side string, price, amount float64) (orderO
 	}
 	var rawRes rawOrder
 	err = json.Unmarshal(r, &rawRes)
+	if err != nil {
+		return
+	}
+	if !rawRes.Success {
+		err = errors.New(string(r))
+		return
+	}
+	orderOid = rawRes.Data.OrderOid
+	return
+}
+
+// CreateOrderByString is used to create order at Kucoin along with other meta data. This ByString version is fix precise problem.
+func (b *Kucoin) CreateOrderByString(symbol, side string, price, amount string) (orderOid string, err error) {
```

Comment:
Nice! Remove "string" type after "side" arg. declaration, it just unneeded in case of all string args.

Reply:
ok, what should i do? i am a submission beginner.

Reply:
Change method arguments from:
```code 
symbol, side string, price, amount string
```
to:
```code
symbol, side, price, amount string
```

Reply:
And after, new commit with fixed code. It will automatically update pull request.